CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,55 @@
+# Code of Conduct
+To be a truly great community, SwiftNIO needs to welcome developers from all walks of life,
+with different backgrounds, and with a wide range of experience. A diverse and friendly
+community will have more great ideas, more unique perspectives, and produce more great
+code. We will work diligently to make the SwiftNIO community welcoming to everyone.
+
+To give clarity of what is expected of our members, SwiftNIO has adopted the code of conduct
+defined by [contributor-covenant.org](https://www.contributor-covenant.org). This document is used across many open source
```

Comment:
Are we trying to keep lines a certain max width? If so, then we should break the line in between "is" and "used" here, and adjust the next line accordingly.

Reply:
ok